usage/rest-server/src/main/java/brooklyn/rest/resources/LocationResource.java

```diff
@@ -82,7 +86,22 @@ public LocationSummary apply(LocationDefinition l) {
         return FluentIterable.from(brooklyn().getLocationRegistry().getDefinedLocations().values())
                 .transform(transformer)
                 .filter(LocationSummary.class)
-                .toSortedList(SummaryComparators.displayNameComparator());
+                .toSortedList(nameOrSpecComparator());
+    }
+
+    private static NaturalOrderComparator COMPARATOR = new NaturalOrderComparator();
+    private static Comparator<LocationSummary> nameOrSpecComparator() {
+        return new Comparator<LocationSummary>() {
+            @Override
+            public int compare(LocationSummary o1, LocationSummary o2) {
+                return COMPARATOR.compare(getNameOrSpec(o1).toLowerCase(), getNameOrSpec(o2).toLowerCase());
+            }
+        };
+    }
+    private static String getNameOrSpec(LocationSummary o) {
```

Comment:
would this be better in the comparator as a non static method? Or is it more widely useful?